src/qml/qmlconfigproxy.cpp

```diff
@@ -1,8 +1,11 @@
 #include "qml/qmlconfigproxy.h"
 
+#include <qvariant.h>
+
```

Comment:
I see no other changes to this file other than the new includes -- is there a reason these were added?

Reply:
I'll double check but I think this related to removing includes from other header files, which this one was indirectly using